src/testing/workspace-fixtures.ts

```diff
@@ -41,6 +41,10 @@ export const protectedAzureWorkspace: AzureWorkspace = _.merge(defaultAzureWorks
   policies: [protectedDataPolicy],
 });
 
+export const protectedPhiTrackingAzureWorkspace: AzureWorkspace = _.merge(defaultAzureWorkspace, {
+  policies: [protectedDataPolicy, phiTrackingPolicy],
+});
```

Comment:
Nit: Is it possible to use vanilla JS here instead of Lodash?

```suggestion
export const protectedPhiTrackingAzureWorkspace: AzureWorkspace = {
  ...defaultAzureWorkspace,
  policies: [protectedDataPolicy, phiTrackingPolicy],
};
```

Reply:
Done.